mailbox/event/event-memory/src/main/java/org/apache/james/mailbox/events/InVMEventBus.java

```diff
@@ -53,6 +55,7 @@ public InVMEventBus(EventDelivery eventDelivery, RetryBackoffConfiguration retry
         this.eventDeadLetters = eventDeadLetters;
         this.registrations = Multimaps.synchronizedSetMultimap(HashMultimap.create());
         this.groups = new ConcurrentHashMap<>();
```

Comment:
now that it's protected by a lock and initialized only once, you could define that as an `Optional<ImmutableMap>`, right?